dodo.py

```diff
@@ -32,12 +61,16 @@ def merge():
 
             with open(OUTPUT_FILE, "w+", encoding="utf-8") as outfile:
                 for file in glob.glob(INPUT_FILES):
-                    print(file)
                     with open(file, encoding="utf-8") as infile:
-                        outfile.write(infile.read())
+                        input = infile.read()
+                        # remove curly quotes, not expected in command line text
+                        import re
+
+                        changed_input = re.sub("[“„“]", "'", input)
```

Comment:
The 2nd `“` in the regex shouldn't be necessary.

Reply:
Interesting - it might be a font issue here, but they are actually different symbols. In Unicode:
[U+201C](https://www.fileformat.info/info/unicode/char/201c/index.htm)
[U+201D](https://www.fileformat.info/info/unicode/char/201d/index.htm)
[U+201E](https://www.fileformat.info/info/unicode/char/201e/index.htm)

I've added a comment about this for future readers

Reply:
Oh weird, yeah I see two left double quotes: `“„“`, not `“„”`.